deploy/chart/values.yaml

```diff
@@ -37,3 +37,6 @@ tolerations:
   - key: node.kubernetes.io/unreachable
     operator: Exists
     effect: NoSchedule
+env:
+  - name: EXAMPLE_ENV_VAR
```

Comment:
Should this be commented out since it's an example?

Reply:
yeah, just had that thought when looking it over